lib/Widgets/Utils.vala

```diff
@@ -152,27 +152,32 @@ public static string accel_to_string (string accel) {
  *
  * @return {@link Pango} markup with the description label on one line and a list of human-readable accels on a new line
  */
-public static string markup_accel_tooltip (string[] accels, string? description = null) {
-    string[] unique_accels = {};
+public static string markup_accel_tooltip (string[]? accels, string? description = null) {
+    string[] parts = {};
+    if (description != null && description != "") {
+        parts += description;
+    }
+    if (accels != null &&  accels.length > 0) {
+        string[] unique_accels = {};
 
-    for (int i = 0; i < accels.length; i++) {
-        var accel_string = accel_to_string (accels[i]);
+        for (int i = 0; i < accels.length; i++) {
+            var accel_string = accel_to_string (accels[i]);
 
-        if (!(accel_string in unique_accels)) {
-            unique_accels += accel_string;
+            if (!(accel_string in unique_accels)) {
+                unique_accels += accel_string;
+            }
         }
-    }
 
-    ///TRANSLATORS: This is a delimiter that separates two keyboard shortcut labels like "⌘ + →, Control + A"
-    var accel_label = string.joinv (_(", "), unique_accels);
+        ///TRANSLATORS: This is a delimiter that separates two keyboard shortcut labels like "⌘ + →, Control + A"
+        var accel_label = string.joinv (_(", "), unique_accels);
 
-    var markup = """<span weight="600" size="smaller" alpha="75%">%s</span>""".printf (accel_label);
+        var markup = """<span weight="600" size="smaller" alpha="75%">%s</span>""".printf (accel_label);
 
-    if (description != null && description != "") {
-        markup = string.join ("\n", description, markup);
+        var accel_markup = """<span weight="600" size="smaller" alpha="75%">%s</span>""".printf (accel_label);
```

Comment:
Am I missing something or is this a duplicate of the line above. It looks like the variable `markup` is never used in this branch

Reply:
Nop, that's definitely duplicate. I don't know how I've missed that. Maybe got lost in a merge conflict.